src/server.js

```diff
@@ -10,7 +11,7 @@ const logger = require('./utils/logger');
 
 
 // Config
-const SERVER_PORT = process.env.PORT || config.get('SERVER.PORT');
+const SERVER_PORT = process.env.PORT;
 const API_BASE = '/api/v1';
```

Comment:
May be removed, used in a single place as far as i know of. Can be replaced with a string.